src/components/app/layout/Header.module.scss

```diff
@@ -9,11 +9,29 @@
   .logo-container {
     width: 90px;
     height: 30px;
+    margin-left: 50px;
 
     .navbar-logo {
       width: 100%;
       height: 100%;
       object-fit: contain;
     }
   }
+  .links-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    margin-right: 50px;
+
+    .links {
+      color: #ffffff;
+      font-family: sans-serif;
+      list-style: none;
+      margin-right: 40px;

```

Comment:
we need to make sure that the last link item doesn't get margin right
you can use &::not(:last-child) {} and add the margin inside the curly braces to fix that, and after doing it please test it by **inspect element** to make sure it's applied correctly